.env.example

```
@@ -1 +1,8 @@
 VITE_API_BASE_URL=https://mail-api.ajou.app
+VITE_FIREBASE_API_KEY=AIzaSyDhKHTOFbfXukC1xbByXKrKXMX6OzrhJbo
+VITE_FIREBASE_AUTH_DOMAIN=mailsangja.firebaseapp.com
+VITE_FIREBASE_PROJECT_ID=mailsangja
+VITE_FIREBASE_STORAGE_BUCKET=mailsangja.firebasestorage.app
+VITE_FIREBASE_MESSAGING_SENDER_ID=firebase_sender_id
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_FIREBASE_MESSAGING_SENDER_ID key should go before the VITE_FIREBASE_PROJECT_ID key

(UnorderedKey)